pipeline.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright (C) 2019 Zilliqa
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+# This script is dedicated for CI use
+#
+
+set -x
+set -e
+
+export scilla_container_id=$(docker run -d --name zilliqa_scilla_$HOSTNAME zilliqa/scilla:v0.11.0 sleep infinity)
```

Comment:
Are we sure we'll be using Scilla v 11.0 for master?    I thought it was more for 8.2.  Maybe that's correct, just pointing out that it would make sense to extract it into some variable so that configuration becomes more obvious.

Reply:
I implemented in Jenkins just a replica of the current process in Travis. I am not sure of what should be the right version, but a good time to change if needed.

Reference: https://github.com/Zilliqa/Zilliqa/blob/master/.travis.yml#L61

Reply:
The current build of evm does infact use version 0.11.0, this caused some issues when we moved to ubuntu 20.04 as scilla team scrambled to get some imprtant fixes from the latest version into 0.11.0 ubuntu version 20.04.

I will add a ticket into jira for investigating upgrading to what scilla team have in production and make sure we are correct..